Formula/b/bozohttpd.rb

```diff
@@ -0,0 +1,44 @@
+class Bozohttpd < Formula
+  desc "Small and secure http version 1.1 server"
+  homepage "http://www.eterna.com.au/bozohttpd/"
+  url "http://www.eterna.com.au/bozohttpd/bozohttpd-20220517.tar.bz2"
+  sha256 "9bfd0942a0876e5529b0d962ddbcf50473bcf84cf5e4447043e4a0f4ea65597a"
+  license "BSD-2-Clause"
+
+  depends_on "pkg-config" => :build
+  depends_on "rtmpdump" => [:build, :test]
```

Comment:
I don't see any usage of `rtmpdump` in the test, should it be a runtime dependency or only a buildtime?